src/app/components/crypto-assets/components/asset-row-grid.tsx

```diff
@@ -5,14 +5,28 @@ interface AssetRowGridProps {
   balance: React.ReactNode;
   caption: React.ReactNode;
   usdBalance?: React.ReactNode;
+  connectBtn?: React.ReactNode;
```

Comment:
On naming: all this really does is place an element to the right. Maybe in future it isn't a connect btn. Could be named on where it puts "unknown" content, than assuming what the content will always be.

Reply:
rightElement?